lib/types/reducers.js

```diff
@@ -281,6 +281,7 @@ export type DateTimeFilter = {
 
 export type FilterState = {
   activeFeeds: any,
+  comparedVersion: any,
```

Comment:
Is it possible to use a more specific type here?

Reply:
Updated in https://github.com/ibi-group/datatools-ui/pull/528/commits/4b1a300ae1e2f573081c305d6b9ddc7cb418ab69.